src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -360,11 +365,11 @@ class FileField extends React.Component {
                       onSubmitPassword={this.onSubmitPassword}
                     />
                   )}
-                  {!file.uploading && (
-                    <div>
+                  {showButtons && (
+                    <div className="vads-u-margin-top--2">
                       <button
                         type="button"
-                        className="va-button-link"
+                        className="usa-button-secondary vads-u-width--auto delete-file"
```

Comment:
Is the `delete-file` class only added for testing reasons? If so, can the test reliable use some other way to determine this button's presence/absence?

Reply:
Yes, for testing reasons. The `usa-button-secondary` class is also applied to the "Add' button. If the class shouldn't be added, I can use the width-auto class that I added, but that doesn't seem as suitable.

Reply:
Duh, I forgot the add button isn't an actual button. So I removed the extra class. This should be all set!